ui/src/shared/utils/errors.ts

```diff
@@ -12,6 +13,19 @@ if (CLOUD) {
   })
 }
 
+export const reportSingleErrorMetric = (
+  tags: PointTags = {},
+  fields: PointFields = {},
+  measurement = 'ui_error'
+) => {
+  if (CLOUD) {
+    const points = {
+      points: [{measurement, tags, fields: {errorCount: 1, ...fields}}],
+    }
+    reportPoints(points)
```

Comment:
why didn't you use reportEvent here? it comes with free point pooling

Reply:
@drdelambre i didn't want to worry about passing in timestamps

Reply:
@hoorayimhelping i did this so that i didn't have to worry about timestamps
https://github.com/influxdata/influxdb/pull/18576/files#diff-a8c1840ed70957a404f34926551d6ad8R8